app/models/school.rb

```diff
@@ -81,6 +81,12 @@ def self.requiring_a_new_computacenter_reference
     gias_status_open.where(computacenter_change: %w[new amended]).or(gias_status_open.where(computacenter_reference: nil))
   end
 
+  def human_for_school_type
+    I18n.t(school_type, scope: %i[activerecord attributes school school_type], default: 'Other')
```

Comment:
@asmega Why do we need the rescue here if we set a default with `I18n.t`?

Reply:
its when `school_type` is `` or empty string from `School#school_type` let me try making that return nil to see if the tests pass